shell/packages/sandstorm-db/migrations.js

```diff
@@ -695,12 +726,14 @@ function migrateToLatest() {
 
   } else {
     const applied = Migrations.findOne({ _id: "migrations_applied" });
+    let isNewServer = false;
     let start;
     if (!applied) {
       // Migrations table is not yet seeded with a value.  This means it has
       // applied 0 migrations.  Persist this.
       Migrations.insert({ _id: "migrations_applied", value: 0 });
       start = 0;
+      isNewServer = true;
```

Comment:
If the server crashes in the middle of running migrations but before running new-server setup then when it comes back up it'll think it is not a new server and won't do the setup.